controllers/infinispan.go

```diff
@@ -108,18 +109,16 @@ func (r *InfinispanReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Ma
 		r.log.Info("Defaults:", "Annotations", defaultAnnotations, "Labels", defaultLabels)
 	}
 	return ctrl.NewControllerManagedBy(mgr).
-		For(&infinispanv1.Infinispan{}).
+		For(&infinispanv1.Infinispan{}, builder.WithPredicates(predicate.Or(predicate.GenerationChangedPredicate{}, predicate.AnnotationChangedPredicate{}, predicate.LabelChangedPredicate{}))).
 		Owns(&corev1.ConfigMap{}).
-		Owns(&corev1.Secret{}).
+		Owns(&corev1.Secret{}, builder.WithPredicates(predicate.Funcs{CreateFunc: func(ce event.CreateEvent) bool { return false }})).
```

Comment:
This is almost impossible to read :smile: 

Can you update the formatting to be like:

```go
		For(&infinispanv1.Infinispan{},
			builder.WithPredicates(
				predicate.Or(
					predicate.GenerationChangedPredicate{},
					predicate.AnnotationChangedPredicate{},
					predicate.LabelChangedPredicate{},
				),
			),
		).
		Owns(&corev1.ConfigMap{}).
		Owns(&corev1.Secret{},
			builder.WithPredicates(
				predicate.Funcs{
					CreateFunc: func(_ event.CreateEvent) bool { return false }},
			),
		).
```